src/routes/(protected)/+layout.svelte

```
@@ -8,6 +8,9 @@
     import "@fontsource/merriweather/latin.css";
     import oxanium400Woff2 from "@fontsource/oxanium/files/oxanium-latin-400-normal.woff2?url";
     import { afterNavigate, beforeNavigate } from "$app/navigation";
+
+    // Save scroll positions per URL for back-navigation restore
+    const scrollPositions = new Map<string, number>();
```

ESLint

[error] Found a mutable instance of the built-in Map class. Use SvelteMap instead.

(svelte/prefer-svelte-reactivity)